gaia-switch.js

```diff
@@ -133,8 +146,13 @@ module.exports = component.register('gaia-switch', {
         this.els.handle.style.transform = '';
         this.els.handle.style.transition = '';
 
-        if (value) { this.setAttr('checked', ''); }
-        else { this.removeAttr('checked'); }
+        if (value) {
+          this.setAttr('checked', '');
```

Comment:
`.setAttr()` is a special version of `setAttribute()` provided by `gaia-component.js` base-class. It sets attrs inside shadow-root and on `:host`.